src/api/trace/span.rs

```diff
@@ -152,8 +152,15 @@ pub trait Span: fmt::Debug + 'static + Send + Sync {
     /// Calls to `end` a Span MUST not have any effects on child `Span`s as they may
     /// still be running and can be ended later.
     ///
-    ///This API MUST be non-blocking.
+    /// This API MUST be non-blocking.
     fn end(&self);
```

Comment:
It may be useful to implement this method by default in terms of `end_with_timestamp` now to avoid requiring redundant impls for both methods.